XIVSlothCombo/Combos/RDM.cs

```diff
@@ -843,36 +1058,47 @@ const int
                 int white = gauge.WhiteMana;
                 int blackThreshold = white + IMBALANCE_DIFF_MAX;
                 int whiteThreshold = black + IMBALANCE_DIFF_MAX;
-                var actionIDCD = GetCooldown(actionID);
-                if (IsEnabled(CustomComboPreset.RedMageOgcdComboOnCombos) && actionIDCD.IsCooldown && IsOffCooldown(RDM.Fleche) || IsEnabled(CustomComboPreset.RedMageOgcdComboOnCombos) && actionIDCD.IsCooldown && IsOffCooldown(RDM.ContreSixte))
-                {
-                    if (level >= RDM.Levels.ContreSixte && level <= RDM.Levels.Fleche)
-                        return CalcBestAction(actionID, RDM.ContreSixte, RDM.Fleche);
 
-                    if (level >= RDM.Levels.ContreSixte && level <= RDM.Levels.Fleche)
-                        return CalcBestAction(actionID, RDM.ContreSixte, RDM.Fleche);
+                var canWeave = CanWeave(actionID);
```

Comment:
If you use it once, you shouldn't make a var for it IMO.

Reply:
I use it 3 times actually, did the var to just calculate it once per invocation, did that because I dont know the cost of that

Reply:
I didn't see the other two times with Ctrl f haha no worries